doc/architecture/decisions/0007-relationships.md

```diff
@@ -8,42 +8,44 @@ We need to be able to collect the (one way) relationships between list items. Fo
 
 ### URLs
 
-The existing url design for lists provides an approach to asking questions related to a given list item:
+The url for all relationships for a given list item would be:
 
 ```
-/<list_identifier>/<list_item_id>/<block_id>
+/<block_id>/<from_list_item_id>
```

Comment:
As discussed. I'm not sure what this URL now represents. Is it a summary, and if so how does that apply to relationships.